homeassistant/components/shelly/strings.json

```
@@ -118,5 +118,11 @@
         }
       }
     }
+  },
+  "issues": {
+    "push_update_failure": {
+      "title": "Shelly device {device_name} push update failure",
+      "description": "Push updates from the Shelly device {device_name} with IP address {ip_address} do not reach the Home Assistant server. You need to check CoIoT configuration in the web panel of the device and your network configuration."
```

Small suggestion to simplify the message. Also wondering if we need to mention checking your network configuration or if we can just say Check the CoIoT configuration in the device's web panel?

```suggestion
      "description": "Home Assistant is not receiving push updates from the Shelly device {device_name} with IP address {ip_address}. Check the CoIoT configuration in the web panel of the device and your network configuration."
```

---

The problem can also be caused by network configuration, e.g. Shelly device in a separate VLAN or blocking multicast on the firewall of the device on which HA is running, I think that we should leave information about checking the network configuration.